lib/public/AppFramework/Http/Response.php

```diff
@@ -201,6 +205,7 @@ public function getCookies() {
 	 * @param string $value The value, null will delete it
 	 * @return $this
 	 * @since 6.0.0 - return value was added in 7.0.0
+	 * @depreacted Use setHeaders instead
```

Comment:
```suggestion
	 * @deprecated Use setHeaders instead
```
The same typo is present several times in the PR